File: easy/SmallerNumbersThanCurrent.cs
public static class SmallerNumbersThanCurrent {
	[Challenge("Smaller Numbers Than Current")]
	public static int[] Solve(int[] nums) {
	int[] result = new int[nums.Length];
	for(int i = 0; i < nums.Length; i++) {
		for(int j = 0; j < nums.Length; j++) {
			if(j != i && nums[i] > nums[j]) {
				result[i]++;
			}
		}
	}
		return result;
	}
}

File: src/easy/smaller-numbers-than-current.ts
export const smallerNumbersThanCurrent = (nums: number[]): number[] => {
    const result = Array(nums.length).fill(0);
    for (let i = 0; i < nums.length; i++) {
        for (let j = 0; j < nums.length; j++) {
            if (nums[i] > nums[j]) {
                result[i]++;
            }
        }
    }
    return result;
};
